### PR TITLE
Travis Setup

### PR DESCRIPTION
**Pull request by fthaler**
- Basic Travis CI setup based on Docker to support CUDA compilation (Docker images from https://github.com/fthaler/gridtools-docker)
- Asynchronous output in PyUtils to avoid time out on Travis
- Possibility to run PyUtils tests without SLURM

### CODE REVIEW

**Comment by fthaler** [comment]
launch strgrid

**Comment by fthaler** [comment]
launch icgrid

**Comment by fthaler** [comment]
launch strgrid

**Comment by fthaler** [comment]
launch icgrid

**Comment by fthaler** [comment]
launch icgrid

**Comment by fthaler** [comment]
launch strgrid

File: .gitignore [hunk 1]
```diff
@@ -51,3 +51,4 @@ __pycache__
 !Vagrant_bootstrap.sh
 !Vagrantfile
 !version.txt
+!.travis.yml
```
Comment (havogt) [inline .gitignore]:
```suggestion
!.travis.yml

```

**Comment by havogt** [comment]
launch mpistrgrid

**Comment by havogt** [comment]
merge switch to d75

**Comment by havogt** [comment]
launch mpistrgrid

**Pull request merged**